content/index.html.haml

```diff
@@ -39,7 +39,7 @@ homepage: true
             Easy installation
           %p
             Jenkins is a self-contained Java-based program, ready to run
-            out-of-the-box, with packages for Windows, Linux, macOS and other
+            out-of-the-box, with packages for Windows, Linux, macOS, and other
```

Comment:
this isn't a grammatical error is this a writing style, doesn't make any sense to change it randomly in my opinion

Reply:
I agree with @timja.  The use of the Oxford comma has long been disputed among various groups of writers.  Changing it in this one location is not a helpful change.  I question the value of changing it throughout the documentation as well.